core/testing/src/commonMain/kotlin/io/github/droidkaigi/confsched/testing/robot/core/FontScaleRobot.kt

```diff
@@ -0,0 +1,16 @@
+package io.github.droidkaigi.confsched.testing.robot.core
+
+import dev.zacsweers.metro.Inject
+
+interface FontScaleRobot {
+    fun setFontScale(fontScale: Float)
+}
+
+expect fun setPlatformFontScale(fontScale: Float)
```

Comment:
- I am using last year's FontScaleRobot almost as is.
- https://github.com/DroidKaigi/conference-app-2024/blob/57c38a76beb5b75edc9220833162e1257f40ac06/core/testing/src/main/java/io/github/droidkaigi/confsched/testing/robot/MiniRobots.kt#L173-L181